readthedocs/config/config.py

```diff
@@ -263,6 +263,12 @@ def validate(self):
     def using_build_tools(self):
         return isinstance(self.build, BuildWithTools)
 
+    @property
+    def is_using_conda(self):
+        if self.using_build_tools:
+            return self.python_interpreter in ("conda", "mamba")
+        return self.conda
```

Comment:
Shouldn't this be like this?

```suggestion
        return self.conda is not None
```